### PR TITLE
add selectable callback parameter

### PR DESCRIPTION
**Pull request by azzra**

### CODE REVIEW

File: src/angular-moment-picker.js [hunk 1]
```diff
@@ -62,20 +62,22 @@
...
+				maxView:    '@?',
+				startView:  '@?',
+				minDate:    '=?',
+				maxDate:    '=?',
+				disabled:   '=?disable',
+				autoclose:  '=?',
+				today:      '=?',
+				keyboard:   '=?',
+				change:     '&?',
+				selectable: '@?'
```
Comment (indrimuska) [inline src/angular-moment-picker.js]: This is an optional callback function, so you should use the `"&?"` binding.

File: src/angular-moment-picker.js [hunk 2]
```diff
@@ -169,7 +171,10 @@
 					return !angular.isDefined($scope.maxDateMoment) || value.isBefore($scope.maxDateMoment, precision) || value.isSame($scope.maxDateMoment, precision);
 				},
 				isSelectable: function (value, precision) {
-					return $scope.limits.isAfterOrEqualMin(value, precision) && $scope.limits.isBeforeOrEqualMax(value, precision);
+					var selectable = angular.isFunction($scope.$parent[$scope.selectable]) ? $scope.$parent[$scope.selectable] : function () {
```
Comment (indrimuska) [inline src/angular-moment-picker.js]: Keep in mind that you should never use `$parent` scope. The directive uses a private scope, so you should care only about your scope (`$scope`).

When you define an attribute using `&` binding ([read more](https://docs.angularjs.org/api/ng/service/$compile#-scope-)) you can call the property directly specifying any attributes you want to pass through to your callback as an object. In other terms, you should update the code like this:

```javascript

isSelectable: function (value, precision) {
	var selectable = angular.isFunction($scope.selectable) ? $scope.selectable({ value: value, precision: precision }) : true;
	return $scope.limits.isAfterOrEqualMin(value, precision) && $scope.limits.isBeforeOrEqualMax(value, precision) && selectable;
}
```

Comment (azzra) [inline src/angular-moment-picker.js]: Changes made as requested. I added some doc too.

**Comment by indrimuska** [comment]
Thank you @azzra for the good work! 👍

**Comment by azzra** [comment]
You're welcome. I made a typo in the readme: `in you controller` -> `in your controller`

**Comment by indrimuska** [comment]
Don't worry, I'm going to reformat it, adding a new "Methods" section.

**Pull request merged**